src/cljs/org/broadinstitute/firecloud_ui/page/workspace/summary/tab.cljs

```diff
@@ -153,7 +153,7 @@
                                                               :defaultValue description
                                                               :style {:width 400}
                                                               :rows 5}))
-                description description
+                description [:div {:style {:whiteSpace "pre"}} description]
```

Comment:
Yes, thanks - for the briefest of moments last night, I thought my ancient knowledge was useful :-)